plugins/redis/plugin.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2018-present the CoreDHCP Authors. All rights reserved
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package redisplugin
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"strings"
+	"time"
+
+	"github.com/coredhcp/coredhcp/handler"
+	"github.com/coredhcp/coredhcp/logger"
+	"github.com/coredhcp/coredhcp/plugins"
+	"github.com/gomodule/redigo/redis"
+	"github.com/insomniacslk/dhcp/dhcpv4"
+	"github.com/insomniacslk/dhcp/dhcpv6"
+)
+
+// various global variables
+var log = logger.GetLogger("plugins/redis")
+var pool *redis.Pool
+var LeaseTime time.Duration
```

Comment:
Does this need to be exported? If not start it with lower case.

Reply:
Oops...I have changed it to lowercase.